community/pdns-recursor/APKBUILD

```diff
@@ -1,22 +1,24 @@
 # Contributor: Olivier Mauras <olivier@mauras.ch>
 # Maintainer: tcely <pdns-recursor+aports@tcely.33mail.com>
-pkgname=pdns-recursor
-pkgver=4.2.0
+_pkgname="pdns-recursor"
+pkgname="${_pkgname}"
```

Comment:
why is the `_pkgname` variable name introduced?

Reply:
It is a convenient way to distinguish between upstream name and the Alpine Linux package name.